sphinx_rtd_theme/__init__.py

```diff
@@ -23,6 +34,14 @@ def get_html_theme_path():
 def setup(app):
     app.add_html_theme('sphinx_rtd_theme', path.abspath(path.dirname(__file__)))
 
+    # Warn if a prerelease version of the theme is used
+
+    parsed_version = pkg_resources.parse_version(__version__)
+    if parsed_version.is_prerelease:
+        logger.warning(_('You are using a pre-release version (%s) of sphinx-rtd-theme.'), __version__)
+        logger.warning(_('Use a stable release if you encounter problems.'))
+        logger.warning(_('See <https://sphinx-rtd-theme.readthedocs.io/en/stable/installing.html>.'))
```

Comment:
I think the URL is unnecessary I think a user who may have installed a dev version already knows how to install the theme

Reply:
I agree it is a bit much, but I want to make this as actionable as possible.